internal/testing/check_api_change.sh

```diff
@@ -31,19 +31,25 @@ set -euo pipefail
 UPSTREAM_BRANCH="${TRAVIS_BRANCH:-master}"
 echo "Checking for incompatible API changes relative to ${UPSTREAM_BRANCH}..."
 
-go install -mod=readonly golang.org/x/exp/cmd/apidiff
-
+INSTALL_DIR="$(mktemp -d)"
 MASTER_CLONE_DIR="$(mktemp -d)"
 PKGINFO_BRANCH=$(mktemp)
 PKGINFO_MASTER=$(mktemp)
 
 function cleanup() {
+  rm -rf "$INSTALL_DIR"
   rm -rf "$MASTER_CLONE_DIR"
   rm -f "$PKGINFO_BRANCH"
   rm -f "$PKGINFO_MASTER"
 }
 trap cleanup EXIT
 
+# Move to a temporary directory while installing apidiff to avoid changing
+# the local .mod file.
```

Comment:
Nit: point to most relevan golang issue here, so we can track if this ever gets resolved?

Reply:
I can't find one that is open :-(.